content/sensu-go/6.3/operations/deploy-sensu/generate-certificates.md

```diff
@@ -290,12 +290,125 @@ Reopen the root CA certificate, expand Trust, select Always Trust, and save your
 Delete the root CA certificate from login.
 {{< /code >}}
 
-{{< code powershell "Windows" >}}
-TODO: Document steps for adding CA root to Windows trust store
+{{< /language-toggle >}}
+
+## Renew self-generated certificates
+
+To keep your Sensu deployment running smoothly, renew your self-generated certificates before they expire.
+Depending on how your certificates are configured, one backend certificate may expire before the others or all three backend certificates may expire at the same time.
+The agent certificate also expires.
+
+This section explains how to find certificate expiration dates, confirm whether certificates have already expired, and renew certificates.
+
+### Find certificate expiration dates
+
+Use this check to find certificate expiration dates so you can renew certificates before they expire and avoid observability interruptions:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: expired_certs
+  namespace: default
+spec:
+  command: openssl x509 -noout -enddate -in cert.pem
+  subscriptions:
+  - system
+  publish: true
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "expired_certs"
+  },
+  "spec": {
+    "command": "openssl x509 -noout -enddate -in cert.pem.sh",
+    "subscriptions": [
+      "system"
+    ],
+    "publish": true
+  }
+}
 {{< /code >}}
 
 {{< /language-toggle >}}
 
+The check output will be in the format `notAfter=Month  Day HH:MM:SS Year Timezone`.
+For example:
+
+{{< code shell >}}
+notAfter=Jul  3 22:23:50 2021 GMT
+{{< /code >}}
+
+Add a [handler][17] to send the check output as a notification or to a log file.
```

Comment:
Maybe we could just add a handler that prints to a file, like the debug handler?